src/common/utils.js

```diff
@@ -278,6 +278,16 @@ export const formatSpeed = (bytes, milliseconds) => humanFormat(
   { scale: 'binary', unit: 'B/s' }
 )
 
+export const parseDuration = seconds => {
```

Comment:
Move this directly with `FormattedDuration`, we'll move it back in `utils` if we need it somewhere else.